Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -4552,8 +4552,12 @@ void WebPage::getAccessibilityTreeData(CompletionHandler<void(const std::optiona
         auto stream = adoptCF(CFWriteStreamCreateWithAllocatedBuffers(0, 0));
         CFWriteStreamOpen(stream.get());
 
-        CFWriteStreamWrite(stream.get(), treeData->liveTree.utf8().dataAsUInt8Ptr(), treeData->liveTree.utf8().length());
-        CFWriteStreamWrite(stream.get(), treeData->isolatedTree.utf8().dataAsUInt8Ptr(), treeData->isolatedTree.utf8().length());
+        auto liveTreeUTF8 = treeData->liveTree.utf8();
+        auto liveTreeUTF8Span = liveTreeUTF8.span();
+        CFWriteStreamWrite(stream.get(), liveTreeUTF8Span.data(), liveTreeUTF8Span.size());
```

Comment:
I would probably make a helper function or lambda to avoid wiring this three line thing out twice.